### PR TITLE
Changes for issue #18 (Summary statics in C project don't include the…

### PR DESCRIPTION
**Pull request by mpujari**
Changes for issue #18

### CODE REVIEW

**Comment by dianedownie** [comment]
Thanks for the quick turn around.  This fix looks like this change could impact more than just C language.  Is that the case? Also, do you have a unit test to add to the project as well?

**Comment by mpujari** [comment]
The changes would effect C language but only HEADER files of them. And about unit test, I missed to add them, will do that.

File: src/test/java/com/blackducksoftware/ohcount4j/OhcountTest.java [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+import java.util.List;
+
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+public class OhcountTest extends AbstractOhcount4jTest {
+
+	@Test
+	public void testSummary() throws Exception {
+		String sourceCodePath = getSourceCodePath("cStyle-issue-18");
```
Comment (PDegenPortnoy) [inline src/test/java/com/blackducksoftware/ohcount4j/OhcountTest.java:29]: Inconsistent indentation

File: src/main/java/com/blackducksoftware/ohcount4j/SourceFileUtils.java [hunk 2]
```diff
@@ -45,21 +45,32 @@ public static String head(SourceFile sourceFile) throws IOException {
...
                 return srcFile.head(HEAD_SIZE);
             }
-        } else if (sourceFile.getContents() != null) {
-            // the source is not from the file, so we got to have content initialized
-            return new String(sourceFile.getContents());
         } else {
-            return null;
+        	char contents[] = sourceFile.getContents() != null ? sourceFile.getContents() : new char[0];
+        	// the source is not from the file, so we got to have content initialized
+        	return new String(contents);
```
Comment (PDegenPortnoy) [inline src/main/java/com/blackducksoftware/ohcount4j/SourceFileUtils.java:51]: This looks nearly identical to the code in the `AbstractExtnResolver.getContents` method in #15.

This seems to be a more logical place for this logic, this being a Source File Utility. Could this remove the need for that other code in `AbstractExtnResolver.getContents`?

**Comment by mpujari** [comment]
All above feed backs are covered in PR #25. Closing it.

**Pull request closed without merging**